shell/browser/ui/cocoa/electron_ns_window_delegate.mm

```diff
@@ -329,6 +341,17 @@ - (void)windowDidExitFullScreen:(NSNotification*)notification {
   shell_->HandlePendingFullscreenTransitions();
 }
 
+- (void)windowDidFailToExitFullScreen:(NSWindow*)window {
```

Comment:
My understanding is that failing to exit fullscreen just dumps the
window out of fullscreen without an animation, but always still actually sends the expected `windowDidExitFullScreen` notification - did you find something different in local testing? Otherwise I don't think we need to reset state here as it'll be handled there.

Reply:
No, that matches what I saw. I'll remove this handler.